docs/guides/high-availability/examples/generated/keycloak.yaml

```diff
@@ -424,6 +424,7 @@ spec:
     limits:
       cpu: "6"
       memory: "2250M"
+  automountServiceAccountToken: true
```

Comment:
We should not be probably modifying this file. Moreover, `true` is the default so no need to specify that like this.

We should rather have somewhere specified in the HA guide that it must not be set to `false`. Maybe @ahus1 or @pruivo can provide some pointers what would be the best place.